docs/components/Wrapper.tsx

```diff
@@ -3,7 +3,7 @@ import { ThemeProvider } from '../../src';
 
 const ThemeWrapper: React.SFC = ({ children }) => {
   const { theme } = (window as any).context;
-  return <ThemeProvider theme={theme}>{children}</ThemeProvider>;
+  return <ThemeProvider theme={theme} children={children as any} />;
```

Comment:
Not sure why this needs to be changed / rewritten. Both forms are equivalent and having the children as the children is the preference (only exception would be a prop forwarding where it makes more sense than going over destructuring).